source/results/flux_points/index.rst

```diff
@@ -80,6 +78,33 @@ error and upper limit columns are:
 * ``flux_ul`` : Upper limit with confidence level given by ``UL_CONF``
   header keyword.
 
+A row may have a value and any combination of upper limits and errors:
+
+.. code::
+
+   e_ref dnde dnde_err dnde_errp dnde_errn dnde_ul
+   1000.0 1.0 0.1 0.1 0.1 1.16
+   3000.0 1.0 0.1 0.1 0.1 1.16
+
+A ``nan`` value should be used for empty or missing data such as a bin
+for which there is an upper limit but no value and vice versa, e.g.
+
+.. code::
+
+   e_ref dnde dnde_err dnde_ul
+   1000.0 1.0 0.1 nan
+   3000.0 nan nan 1.16
+   
+The ``ul`` column is an optional boolean flag that can be used to
```

Comment:
Maybe call the column `is_ul` meaning "is an upper limit"?
That's more explicit than `ul`, which people might mistake as the ul value column if they don't read the spec carefully.

@woodmd - Just a suggestion, OK to leave as-is if you prefer.